src/coreclr/src/pal/src/init/pal.cpp

```diff
@@ -91,6 +91,17 @@ int CacheLineSize;
 #endif
 #endif
 
+#if defined(__FreeBSD__)
+#include <sys/types.h>
+#include <sys/param.h>
+#endif
+#if HAVE_GETAUXVAL
+#include <sys/auxv.h>
+#endif
+#if HAVE_SYS_SYSCTL_H || defined(__FreeBSD__)
+#include <sys/sysctl.h>
```

Comment:
This seems to be causing a warning with Ubuntu 20.04 (on WSL2, not exactly sure if it's related). I noticed this error in https://github.com/dotnet/runtime/issues/40919; not exactly sure if this is the cause for the whole build failing.
```  In file included from /home/gotos/source/repos/runtime/src/coreclr/src/pal/src/init/pal.cpp:102:
  /usr/include/x86_64-linux-gnu/sys/sysctl.h:21:2: error: "The <sys/sysctl.h> header is deprecated and will be removed." [-Werror,-W#warnings]
  #warning "The <sys/sysctl.h> header is deprecated and will be removed."
   ^
```